packages/api/controllers/nomination.js

```diff
@@ -24,6 +24,19 @@ const getNominationById = async (req, res) => {
   }
 };
 
+const findAllNominataions = async (req, res) => {
+  try {
+    const nominations = await db.Nomination.findAll();
+    if (nominations.length) {
+      return res.status(200).json(nominations);
+    }
+    return res.status(404).send('No nominations found');
```

Comment:
I know we did this somewhere else, but I might be inclined to just return the empty array with a 200 instead of a 404 (which suggests something went wrong). But 🤷